src/main/java/net/atos/zac/zaaksturing/ZaakafhandelParameterBeheerService.java

```diff
@@ -129,4 +159,76 @@ public ZaakbeeindigParameter readZaakbeeindigParameter(final UUID zaaktypeUUID,
                         String.format("No ZaakbeeindigParameter found for zaaktypeUUID: '%s' and zaakbeeindigRedenId: '%d'", zaaktypeUUID.toString(),
                                       zaakbeeindigRedenId)));
     }
+
+    /**
+     * Zaaktype is aangepast, indien geen concept, dan de zaakafhandelparameters van de vorige versie zoveel mogelijk overnemen
+     *
+     * @param zaaktypeUri uri van het nieuwe zaaktype
+     */
+    public void zaaktypeAangepast(final URI zaaktypeUri) {
```

Comment:
Dit is een grote method. Kan deze nog enigzins worden opgespluts in een aantal andere methods met een goed beschrijvende naam?